docs/introduction.md

```diff
@@ -138,6 +138,43 @@ ncn-m001# sat bash
 (CONTAINER_ID) sat-container# sat status
 ```
 
+## SAT in CSM 1.3
```

Comment:
I'd suggest changing the title of this section to the following. In a few years, most customers will be on CSM 1.3 or later, and we won't need to call out the version in this section title. However, if we change the title then, I believe all links to it in other EX documents will break. I think it's ok to change this now since we list the specific CSM version in the first line of text.
```suggestion
## SAT in CSM
```